algorithms/src/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ install(
 
 kokkos_add_library(
   kokkosalgorithms
+  STATIC
```

Comment:
maybe a quick comment why this is always a static would be helpful for future reference